wpilibc/src/main/native/include/frc/controller/PIDController.h

```diff
@@ -0,0 +1,303 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <functional>
+#include <limits>
+
+#include <wpi/mutex.h>
+
+#include "frc/smartdashboard/SendableBase.h"
+
+namespace frc {
+namespace experimental {
+
+// TODO: remove when this class is moved into frc namespace
+using frc::SendableBase;
+using frc::SendableBuilder;
+
+/**
+ * Class implements a PID Control Loop.
+ *
+ * Creates a separate thread which reads the given PIDSource and takes care of
```

Comment:
This class doesn't create a separate thread.

Reply:
Heh. Basically all of that Doxygen comment except the first line is a lie.